### ISSUE TITLE
geom_smooth support

### ISSUE THREAD

**Issue by brunomioto** [wurli/ggborderline#5]
I would like to ask for geom_smooth() support!

**Comment by wurli** [comment]
I agree this would be nice! I'll try and get to it as soon as I can, may not be for a little while though.
